src/test/java/io/confluent/connect/elasticsearch/ElasticsearchClientTest.java

```diff
@@ -331,7 +330,8 @@ public void testDeleteRecord() throws Exception {
     waitUntilRecordsInES(2);
 
     // delete 1
-    SinkRecord deleteRecord = new SinkRecord(TOPIC, 0, Schema.STRING_SCHEMA, "key0", null, null, 3);
+    SinkRecord deleteRecord = new SinkRecord(TOPIC, 0, Schema.STRING_SCHEMA, "key0", null, null, 3,
```

Comment:
none of the below changes are necessary since we are not writing to datastreams

Reply:
Resolved in chat. Keeping the changes because `maybeAddTimestamp()` takes in a long for its `timestamp` parameter and is called regardless of data streams being set.